Source/Particles/Collision/ScatteringProcess.H

```diff
@@ -42,10 +42,10 @@ public:
 
     ~ScatteringProcess() = default;
 
-    ScatteringProcess (ScatteringProcess const&)            = delete;
-    ScatteringProcess& operator= (ScatteringProcess const&) = delete;
-    ScatteringProcess (ScatteringProcess &&)                = default;
-    ScatteringProcess& operator= (ScatteringProcess &&)     = default;
+    // ScatteringProcess (ScatteringProcess const&)            = delete;
+    // ScatteringProcess& operator= (ScatteringProcess const&) = delete;
+    // ScatteringProcess (ScatteringProcess &&)                = default;
+    // ScatteringProcess& operator= (ScatteringProcess &&)     = default;
```

Comment:
It's not great to allow `ScatteringProcess` to be copied since it can hold substantial data from the cross-section files. But this was the quickest solution to avoid the compiler complaining about the copy of `DSMCFunc` at line https://github.com/ECP-WarpX/WarpX/blob/b0ae812c1aa09b954b9c2fa5ba8b0f22a652696f/Source/Particles/Collision/BinaryCollision/BinaryCollision.H#L227

@ax3l how much of a problem do you think this is?

Reply:
Looks like it is actually a big problem since all the MCC tests fail now.